SpatialGDK/Source/SpatialGDKServices/Private/SpatialPackageManager.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialPackageManager.h"
+#include "SpatialGDKServicesConstants.h"
+
+
+DEFINE_LOG_CATEGORY(LogSpatialPackageManager);
+
+#define LOCTEXT_NAMESPACE "FSpatialPackageManager"
+
+FSpatialPackageManager::FSpatialPackageManager() {}
+
+void FSpatialPackageManager::Init() {}
+
+bool FSpatialPackageManager::TryFetchRuntimeBinary(FString RuntimeVersion)
+{
+
+	UE_LOG(LogSpatialPackageManager, Log, TEXT("RUNTIME VERSION %s"), *RuntimeVersion);
+
+	FString RuntimePath = FString::Printf(TEXT("%s/runtime/runtime.exe"), *SpatialGDKServicesConstants::GDKProgramPath);
```

Comment:
Let's make `runtime.exe` into a constant, SpatialGDKServicesContstants should do the job